book/website/collaboration/task-management.md

```diff
@@ -28,16 +28,84 @@ There are different levels at which tasks exist - at the lowest level tasks can
 * Use a verb
 * What is done
 
+### Sorting tasks 
 
-## Systems
+To decide if and when to do a task, you need more information about the task, such as date and if needed, tags. Tags can be related to:
+
+* Priority, for example Eisenhower matrix (urgent/important) 
+* What tools you need for the task (for example internet) 
+* How long you estimate the task will take (minutes/hours/...) 
+* What is the effort involved (high/low) 
+
+### Kanban
+
+Not a system but more a philosophy. Kanban principle, originates from building cars. The idea behind Kanban is to “manage work by balancing demands with available capacity” (Wikipedia). The goal is to not do too many things at the same time, and finish first what you started working on. 
+
+Tasks (but also projects) progress from left to right in time, for example: 
+
+* Backlog/ideas
+* Next
+* Doing
+* Done
+* Dropped
+
+### Doing tasks
+
+* Put the task on your calendar
+* Pomodoro
+* "Eat The Frog"
+
+
+## Systems / project "pulse"
+
+Switching between managing and doing - assuming you are a team member yourself. How often do you check in with the whole team, when do you plan things, when do you evaluate results, etc. There are various 
```

Comment:
the CI test failed because of the etc. 😅 Described here: https://the-turing-way.netlify.app/community-handbook/style.html

Reply:
Should be fixed now I think